# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/PHttpChannel.ipdl
# Commit: e6c12eec1cef
# Full Hash: e6c12eec1cef32987b7a0b79ccf4420ee097b868
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-30 10:00:42
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P4 Remove On[Start|Stop]Request/OnTransportAndData in PHttpChannel, r=mayhemer,nika,necko-reviewers
#   
#   We move OnStartRequest from PHttpChannel to PHttpBackgroundChannel, thus adjusting
#   message-metadata.ini
#   
# ==============================================================================

diff -r d59b42f58f86 -r e6c12eec1cef netwerk/protocol/http/PHttpChannel.ipdl
--- a/netwerk/protocol/http/PHttpChannel.ipdl	Tue Jun 30 00:32:53 2020 +0000
+++ b/netwerk/protocol/http/PHttpChannel.ipdl	Tue Jun 30 00:33:01 2020 +0000
@@ -103,31 +103,6 @@
   async __delete__();
 
 child:
-  async OnStartRequest(nsHttpResponseHead  responseHead,
-                       bool                useResponseHead,
-                       nsHttpHeaderArray   requestHeaders,
-                       HttpChannelOnStartRequestArgs args);
-
-  // Combines a single OnDataAvailable and its associated OnProgress &
-  // OnStatus calls into one IPDL message
-  // This is a duplicate of the message on PHttpBackgroundChannel, and gets
-  // used for multi-pary streams to make synchronization when we get OnStartRequest 
-  // multiple times easier.
-  async OnTransportAndData(nsresult  channelStatus,
-                           nsresult  transportStatus,
-                           uint64_t  offset,
-                           uint32_t  count,
-                           nsCString data);
-
-  // This is a duplicate of the message on PHttpBackgroundChannel, and gets
-  // used for multi-pary streams to make synchronization when we get OnStartRequest 
-  // multiple times easier.
-  async OnStopRequest(nsresult channelStatus,
-                      ResourceTimingStructArgs timing,
-                      TimeStamp lastActiveTabOptimization,
-                      nsHttpHeaderArray responseTrailers,
-                      ConsoleReportCollected[] consoleReport);
-
   // Used to cancel child channel if we hit errors during creating and
   // AsyncOpen of nsHttpChannel on the parent.
   async FailedAsyncOpen(nsresult status);
